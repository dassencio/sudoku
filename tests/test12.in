4 4
 1  _  _  2  _  7  _  5 13  _  8  _  _  _  _  _
 5  _  _  _ 12  1 15  _  _  6  7  3  4 16  _ 10
 4  _  _  _  _  _  _  _ 12  2  _  1  8  _  5  7
 _ 15  _  3  _  _  _  9  _  _ 11  _  _  2  _  _
 _  8  _  6  _ 11  3  _  _  _  _  5  _ 12  4 16
 _  _  5 10  _  _ 12  _  _  _ 13 16  _ 11  _  8
12  _  4  _  _  _  _  _  _ 11  _  _  _  _  9  _
15 11  _ 16  _  _  8  7  _  _  4  6  3  _  _  _
 _  _  _  1  8 14  _  _  3 10  _  _ 13  _  6  5
 _  2  _  _  _  _  7  _  _  _  _  _  _ 10  _  3
13  _ 14  _  9  3  _  _  _ 15  _  _ 11  1  _  _
 6  4  3  _  1  _  _  _  _ 13  2  _ 16  _ 12  _
 _  _  9  _  _ 12  _  _  5  _  _  _  2  _ 16  _
14  5  _ 15  4  _  9 16  _  _  _  _  _  _  _ 12
10  _ 16 12  7 13 11  _  _  1 15  9  _  _  _  6
 _  _  _  _  _ 10  _  8  7  _ 12  _ 15  _  _  1
