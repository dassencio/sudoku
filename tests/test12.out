 1 16 10  2 11  7 14  5 13  4  8 15 12  6  3  9
 5 13  8  9 12  1 15  2 14  6  7  3  4 16 11 10
 4 14  6 11 10 16 13  3 12  2  9  1  8 15  5  7
 7 15 12  3  6  8  4  9 16  5 11 10 14  2  1 13
 2  8  7  6 13 11  3 15 10  9 14  5  1 12  4 16
 3  9  5 10  2  4 12  1 15  7 13 16  6 11 14  8
12  1  4 14 16  5  6 10  8 11  3  2  7 13  9 15
15 11 13 16 14  9  8  7  1 12  4  6  3  5 10  2
 9 12 15  1  8 14  2  4  3 10 16 11 13  7  6  5
16  2 11  8  5  6  7 13  4 14  1 12  9 10 15  3
13 10 14  7  9  3 16 12  6 15  5  8 11  1  2  4
 6  4  3  5  1 15 10 11  9 13  2  7 16  8 12 14
 8  7  9 13 15 12  1  6  5  3 10  4  2 14 16 11
14  5  1 15  4  2  9 16 11  8  6 13 10  3  7 12
10  3 16 12  7 13 11 14  2  1 15  9  5  4  8  6
11  6  2  4  3 10  5  8  7 16 12 14 15  9 13  1
